package.json

```diff
@@ -43,21 +46,23 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@edx/frontend-platform": "7.0.1",
-    "@openedx/frontend-build": "13.0.27",
+    "@edx/frontend-platform": "^8.0.0",
+    "@openedx/frontend-build": "^14.0.3",
     "@openedx/paragon": "22.0.0",
     "axios": "^1.4.0",
     "es-check": "^2.3.0",
     "husky": "2.7.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "semantic-release": "^19.0.3"
+    "semantic-release": "^19.0.3",
+    "ts-jest": "^29.1.2"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^7.0.1",
-    "@openedx/frontend-build": "^13.0.27",
+    "@edx/frontend-platform": "^8.0.0",
+    "@openedx/frontend-build": "^14.0.0",
```

Comment:
These should have been changed to
```
    "@edx/frontend-platform": "^7.0.1 || ^8.0.0",
    "@openedx/frontend-build": "^13.0.27 || ^14.0.0",
```

Any of the package complete deprecation will make the other project break, thus a breaking change. This change should have bump it to version `3.0.0` instead of `2.1.0`

Reply:
@BilalQamar95 sorry for late comment

Reply:
This was supposed to go as a breaking change as per description, it was [reverted immediately](https://github.com/openedx/react-unit-test-utils/pull/23) and [republished as a breaking change](https://github.com/openedx/react-unit-test-utils/pull/24)